src/deploy/project.py

```diff
@@ -214,6 +214,37 @@ def _get_service(service):
 
         return matched_services
 
+
+    def is_release_deployed(self, release, environment_id):
+        """
+        Checks the release_id matches the `deployment:label` tag on the services
+        and tasks within those services. We check that the desiredCount of tasks 
+        matches the running count of tasks.
+        """
+        ecs_services = self.get_ecs_services(release, environment_id)
+
+        deployed_services = []
+        for _, services in sorted(ecs_services.items()):
+            for service in services:
+                tags = service["response"]["tags"]
+                desired_task_count = service["response"]["desiredCount"]
+                service_deployment_label = next(tag["value"] for tag in tags if tag["key"] == "deployment:label")
+                ecs = self._ecs(
+                    region_name=service.get('region_name'),
+                    role_arn=service.get('role_arn')
+                )
```

Comment:
Minor: I would put this outside the loop, to avoid creating the ECS client repeatedly.

Reply:
I need the service to create it.